src/components/PreviewDraft/PreviewDraftLightboxV2.tsx

```diff
@@ -6,216 +6,23 @@
  *
  */
 
-import { useFormikContext } from "formik";
-import { ReactElement, useMemo, useState, ElementType } from "react";
-import { useTranslation } from "react-i18next";
+import { ReactElement, ElementType } from "react";
 import styled from "@emotion/styled";
 import { spacing } from "@ndla/core";
 import { ModalCloseButton, ModalHeader, ModalSizeType, Modal, ModalTrigger, ModalContent } from "@ndla/modal";
-import { IConcept } from "@ndla/types-backend/concept-api";
-import { IArticle } from "@ndla/types-backend/draft-api";
-import { OneColumn } from "@ndla/ui";
-import PreviewConceptComponent from "./PreviewConcept";
-import PreviewDraft from "./PreviewDraft";
-import { learningResourceFormTypeToDraftApiType } from "../../containers/ArticlePage/articleTransformers";
-import { ConceptFormValues } from "../../containers/ConceptPage/conceptInterfaces";
-import { conceptFormTypeToApiType } from "../../containers/ConceptPage/conceptTransformers";
-import { LearningResourceFormType } from "../../containers/FormikForm/articleFormHooks";
-import { useConcept } from "../../modules/concept/conceptQueries";
-import { useLicenses } from "../../modules/draft/draftQueries";
+import { ConceptPreviewProps, PreviewConcept } from "./PreviewConcept";
+import { CompareConceptPreviewProps, PreviewConceptCompare } from "./PreviewConceptCompare";
+import { MarkupPreviewProps, PreviewMarkup } from "./PreviewMarkup";
+import { PreviewVersion, VersionPreviewProps } from "./PreviewVersion";
 
-interface BaseProps {
+export interface PreviewBaseProps {
```

Comment:
Sida de ikke lenger er i samma fila, denne uansett ikke også brukes her, og sida det ene attributtet uansett overskrives i hver av de 4 som "extender" denne igjen, lurer jeg på om det er bryet verdt å eksportere den bare for å beholde extenden. Lurer på om jeg istedet hadde bare skrevet typen fullt ut i de 4 filene det er snakk om (PreviewConcept, PreviewConceptCompare, PreviewMarkup og PreviewVersion). Føler det er hakket ryddigere.